lib/shelley/src/Cardano/Wallet/Shelley/Network/Node.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE PartialTypeSignatures #-}
```

Comment:
Oh — is it really used in this module? 🤔

Reply:
Nope, good catch!